src/components/Farmlands.vue

```diff
@@ -59,7 +60,10 @@
           Your business can now either fully integrate your retail point of sale system or access a secure web portal when accepting Farmlands Card payments anytime, anywhere.
         </p>
         <div class="m-auto max-w-[450px] pt-4 pb-3">
-          <img src="/farmlands-flow.png">
+          <img
+            src="/farmlands-flow.png"
+            alt=""
```

Comment:
Is an empty alt text the same as having alt text in terms of accessibility?

Reply:
@Ray2126 if you supply no alt text, this would read as “slash-farmlands-dash-flow-dot-p-n-g, Unlabelled image” which isn't helpful. I didn't think the image added any context that wasn't written on the page and didn't have an alt description that would add any value.

Adding an empty alt tag skips the image being announced at all on a screen reader.